kivymd/uix/spinner.py

```diff
@@ -220,14 +220,20 @@ class MDSpinner(ThemableBehavior, Widget):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.color = self.theme_cls.primary_color
+        
+        if not self.color:
+            self.color = self.theme_cls.primary_color
+        
+        if self.color == self.theme_cls.primary_color
+            self.theme_cls.bind(primary_color=self._update_color)
+            
         self._alpha_anim_in = Animation(_alpha=1, duration=0.8, t="out_quad")
         self._alpha_anim_out = Animation(_alpha=0, duration=0.3, t="out_quad")
         self._alpha_anim_out.bind(
             on_complete=self._reset,
             on_progress=self._on_determinate_progress,
         )
-        self.theme_cls.bind(primary_color=self._update_color)
```

Comment:
This binding updated the spinner palette when changing the application's color theme.

Reply:
https://user-images.githubusercontent.com/67905454/112613336-fd63dd00-8e45-11eb-8227-162c610defc0.mp4

The binding is still intact for when the user mentions the color to be self.theme_cls.primary_color. If there is no color specified, the default primary color is taken and the binding still remains intact.

The problem which was, as you can see with the MDRaisedButtons in  the video attached, is that the binding forced an update regardless of the original value, leading to very obvious inconsistencies.

Reply:
[The code for above video](https://github.com/shreyash/demos/blob/main/kivy_spinner_palette.py)
The buttons update the entire app's palette and the spinner changes accordingly.

Reply:
@shreyash

https://user-images.githubusercontent.com/16930280/112614814-c46e3d00-8e32-11eb-996c-16d7aae0cdd3.mov

Reply:
@HeaTTheatR 
Can you check if it's not already broken? I cloned the repository, again and I'm having the same error as you.